.github/workflows/linter.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint
+on:
+  push:
+    branches: [main]
+  pull_request:
```

Comment:
I think we can just have:
```
on:
  pull_request:
    branches: [main]
```
because PRs are the main source of contributions here.

Reply:
Done.